qcore/im.py

```diff
@@ -88,3 +90,58 @@ def order_im_cols_df(df, pattern_order=default_pattern_order):
         [adj_cols.append(col) for col in orig_cols if col not in adj_cols]
 
     return df[adj_cols]
+
+
+def order_ims(unsorted_ims, pattern_order=default_pattern_order):
```

Comment:
Might be above your pay grade. But merge this function and the one above to limit the duplication of the code.